docs/install.rst

```diff
@@ -1,115 +1,146 @@
+.. _installing-astropy:
+
 ************
 Installation
 ************
 
-.. _installing-astropy:
+Overview
+========
+
+The first step to installing ``astropy`` is to ensure that you have a Python
+environment which is **isolated** from your system Python installation. This is
+important because ``astropy`` has many dependencies, and you do not want to accidentally
+break your system by installing incompatible versions of these dependencies.
+
+For this installation guide we use the `conda <https://docs.conda.io/en/latest/>`_
+package manager provided by `miniforge <https://github.com/conda-forge/miniforge>`_.
+This is a popular choice and works well, especially for newcomers. It is easy to install
+and use on all platforms and it makes it easy to install the latest Python version. If
+you already have a ``miniforge``-based Python environment then you can skip to
+:ref:`installing-astropy-with-pip`.
+
+Another option for more experienced users is a virtual environment manager such as the
+Python standard library `venv <https://docs.python.org/3/library/venv.html>`_ module.
+There are numerous resources available to help you set up a virtual environment in this
+manner if you choose this option.
+
+.. note::
+   We **do not recommend** using ``astropy`` with an existing `miniconda
+   <https://docs.anaconda.com/miniconda/>`_ or `Anaconda Python
```

Comment:
I don't think miniconda comes with astropy pre-installed, unlike the full Anaconda env. Maybe should be rephrased to say something to the effect of "don't use full Anaconda install or get astropy from the `defaults` channel".

Reply:
Fixed, hopefully.